lib/main.tsx

```diff
@@ -0,0 +1,29 @@
+
+
+import CdeModal from './components/CdeModal';
+import {InitParams} from "./models.ts";
+import {CdeContextProvider} from "./CdeContext.tsx";
+import {createRoot} from "react-dom/client";
+
+export const init = (props: InitParams) => {
```

Comment:
This is the entrypoint of our library.

The idea is that it will create an element in the original page and add our modal to that element.

Should be kept vanilla simple like this to work with any other website (that may or may not be using react)